<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modCalcSidTimeDlg</class>
 <widget class="QWidget" name="modCalcSidTimeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sidereal Time</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="interact">
      <attribute name="title">
       <string>Interactive Mode</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <layout class="QGridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="2" column="0" colspan="3">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>391</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2" rowspan="2">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QPushButton" name="LocationButton">
           <property name="text">
            <string>Greenwich, United Kingdom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="DateLabel">
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QTimeEdit" name="ST"/>
         </item>
         <item row="3" column="1">
          <widget class="QTimeEdit" name="LT"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="SiderealTimeLabel">
           <property name="text">
            <string>Sidereal time:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="LocalTimeLabel">
           <property name="text">
            <string>Local time:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <layout class="QHBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QDateEdit" name="Date"/>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="LocationLabel">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="batch">
      <attribute name="title">
       <string>Batch Mode</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QComboBox" name="ComputeComboBatch">
           <item>
            <property name="text">
             <string>Compute sidereal time</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Compute standard time</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Input Parameters</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="DateLabelBatch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="DateCheckBatch">
            <property name="text">
             <string>Read from input file</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="LocationLabelBatch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Location:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="LocationButtonBatch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Greenwich, United Kingdom</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="LocationCheckBatch">
            <property name="text">
             <string>Read from input file</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="DateBatch"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="HelpLabel">
         <property name="text">
          <string>Specify Local Time in the Input File</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Input file:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KUrlRequester" name="InputFileBatch"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Output file:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KUrlRequester" name="OutputFileBatch"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QPushButton" name="RunButtonBatch">
             <property name="text">
              <string>Compute</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ViewButtonBatch">
             <property name="text">
              <string>View Output...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>465</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>QPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QComboBox</class>
   <extends>QComboBox</extends>
   <header>QComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>Date</tabstop>
  <tabstop>LocationButton</tabstop>
  <tabstop>LT</tabstop>
  <tabstop>ST</tabstop>
  <tabstop>ComputeComboBatch</tabstop>
  <tabstop>DateBatch</tabstop>
  <tabstop>DateCheckBatch</tabstop>
  <tabstop>LocationButtonBatch</tabstop>
  <tabstop>LocationCheckBatch</tabstop>
  <tabstop>InputFileBatch</tabstop>
  <tabstop>OutputFileBatch</tabstop>
  <tabstop>RunButtonBatch</tabstop>
  <tabstop>ViewButtonBatch</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
